### DISCUSSION TITLE
what is academic mail address for a student

### DISCUSSION THREAD

**Discussion by harsh1c** [community/community#70316] [GitHub Education]
### Hi everyone,

![git](https://github.com/community/community/assets/145260945/3c32b7b1-4f4e-4095-bbcf-54b4c28dd6b8)
I got this message from git team

**Comment by itsdarkhost** [comment]
Mail that was provided by the academic organization
Example of Oxford's academic mail:
first.last@unit.uk

**Reply by harsh1c** [reply]
ohkk...thx for the reply :)
